src/components/Modals/MoveItemToCollectionModal/MoveItemToCollectionModal.tsx

```diff
@@ -29,7 +29,8 @@ export default class MoveItemToCollectionModal extends React.PureComponent<Props
           <CollectionDropdown
             value={collection}
             onChange={this.handleChangeCollection}
-            filter={collection => !isTPCollection(collection)}
+            filter={collection => !isTPCollection(collection) && !collection.isPublished}
+            isCollectionPublished={false}
```

Comment:
and here we could pass the object (so it gets more generic and more extensible)
```suggestion
            fetchCollectionParams ={{ isPublished: false }}
```